src/chainparams.cpp

```diff
@@ -1253,6 +1274,50 @@ void CRegTestParams::UpdateLLMQTestParametersFromArgs(const ArgsManager& args, c
     UpdateLLMQTestParameters(size, threshold, llmqType);
 }
 
+void CRegTestParams::UpdateLLMQInstantSendFromArgs(const ArgsManager& args)
+{
+    if (!args.IsArgSet("-llmqinstantsend")) return;
+
+    const auto& llmq_params_opt = GetLLMQ(consensus.llmqTypeInstantSend);
+    assert(llmq_params_opt.has_value());
+
+    std::string strLLMQType = gArgs.GetArg("-llmqinstantsend", std::string(llmq_params_opt->name));
+
+    Consensus::LLMQType llmqType = Consensus::LLMQType::LLMQ_NONE;
+    for (const auto& params : consensus.llmqs) {
```

Comment:
It seems as this loop already implemented several times:
```
         if (auto optLLMQParams = ranges::find_if_opt(Params().GetConsensus().llmqs, 
                                                     [&strLLMQType](const auto& params){return params.name == strLLMQType;})) {
```
in `src/llmq/utils.cpp` and `src/chainparams.cpp`:
```
    Consensus::LLMQType llmqType = Consensus::LLMQType::LLMQ_NONE;
    for (const auto& params : consensus.llmqs) {
        if (params.name == strLLMQType) {
            llmqType = params.type;
        }
    }
```
May be it's time to refactor to a new method:
```
    const Consensus::LLMQParams& GetLLMQ(const std::string& llmqTypeName) const;
```
?